modin/engines/ray/utils.py

```diff
@@ -61,18 +61,24 @@ def _import_pandas(*args):
     import pandas  # noqa F401
 
 
-def initialize_ray():
+def initialize_ray(
+    override_is_cluster=False, override_redis_address=None, override_redis_password=None
+):
     """Initializes ray based on environment variables and internal defaults."""
     import ray
 
-    if threading.current_thread().name == "MainThread":
+    if threading.current_thread().name == "MainThread" or override_is_cluster:
```

Comment:
When is this condition change necessary?

Reply:
added in docstring